src/index.js

```diff
@@ -1,34 +1,50 @@
+/* eslint-disable import/no-mutable-exports */
+/* eslint-disable import/no-cycle */
 import './style.css';
+import { getLocalStorage, setLocalStorage, populateTasks } from './modules/structure.js';
 
-// Arrayb of Tasks
-const tasks = [
-  {
-    description: 'Wash plate',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Bathe',
-    completed: false,
-    index: 2,
-  },
-];
+let tasks = getLocalStorage();
+const listContainer = document.querySelector('.all-tasks');
 
-const listContainer = document.querySelector('.list-div');
-
-// Show tasks
-const showTasks = () => {
-  tasks.forEach((task) => {
-    const listDiv = document.createElement('li');
-    listDiv.innerHTML = `
-    <input type="checkbox" name="task" id="task" >
-    <label>${task.description}</label>
-    `;
-    listDiv.className = 'todo-task';
-    listDiv.setAttribute('draggable', 'true');
-    listContainer.appendChild(listDiv);
+function addTasks(desc, complete = false) {
+  const index = tasks.length + 1;
+  const newTask = { desc, complete, index };
+  tasks = [...tasks, newTask];
+  setLocalStorage();
+}
+function deleteTask(index) {
+  tasks = tasks.filter((task) => task.index !== +index)
+    .map((task, idx) => {
+      task.index = idx + 1;
+      return task;
+    });
+  setLocalStorage();
+}
+function editTask(input, index) {
+  tasks = tasks.map((task) => {
+    if (task.index === +index) {
+      task.desc = input;
+      populateTasks();
+      setLocalStorage();
+    }
```

Comment:
- [ ] Kindly make sure your `edit` task feature is working for a better User Experience to complete the CRUD Operation, as you may notice when trying to edit a task nothing is happening.
- [ ] Also make sure the delete all completed is implemented.
`You may consider using the filter method to filter all the tasks that have completed set to true.`

Reply:
@christianbiring1 Thank you for your feedback. It is much appreciated. I would like to bring the following to your notice:

1. `Also make sure the delete all completed is implemented`: This functionality is a requirement for the next task and not this one. Kindly help me check this and please confirm this.
2. `Kindly make sure your edit task feature is working for a better User Experience to complete the CRUD Operation, as you may notice when trying to edit a task nothing is happening.`: I made a brief video to show you how the edit feature works. I would be grateful if you look into it again and give me feedback on this.

About the disabling of Linters, I would do my best to fix it. I look forward to your reply. Thank you

Reply:
Hi @Wineshuga, 👋🏿  
Everything seems to check out as you said. Keep up the good work 👍🏿